templates/compose/nodebb.yaml

```diff
@@ -0,0 +1,60 @@
+# ignore: false
+# documentation: https://docs.nodebb.org/
+# slogan: A next-generation discussion platform.
+# tags: communication, forums, discussion
+# logo: svgs/nodebb.svg
+# port: 4567
+
+services:
+  nodebb:
+    image: ghcr.io/nodebb/nodebb:latest
+    restart: unless-stopped
+    environment:
+      - SERVICE_FQDN_nodebb_4567
+    volumes:
+      - nodebb-build:/usr/src/app/build
+      - nodebb-uploads:/usr/src/app/public/uploads
+      - nodebb-config:/opt/config
+      - type: bind
+        source: ./.docker/config/setup.json
+        target: /usr/src/app/setup.json
+        isDirectory: false
+        content: |
```

Comment:
Please use env variables or better our magic variables and do not hardcode secrets.

Reply:
NodeBB sadly doesn't have any listed documentation that I can find that uses environment variables. And they require the existence of setup.json. I don't know if this is a problem fully but I created a new entrypoint that overwrites the existing setup.json with the magic variables.